### ISSUE TITLE
 Trait Error: The ‘children’ trait of a VBox instance ... itkwidgets 1.0a16

### ISSUE THREAD

**Issue by kevinsweeney84** [InsightSoftwareConsortium/itkwidgets#556]
Getting `TraitError: The 'children' trait of a VBox instance contains an Instance of a TypedTuple which expected a Widget, not the Viewer at '0x2aaab5d0ad00'.`

when running the code below in Jupyter Lab

```python
import itkwidgets
import ipywidgets
    
view = itkwidgets.view(geometries=None, background=[0.0, 0.0, 0.0])

# Headings
heading = ipywidgets.HTML(value="<h1>Heading</h1>")

# Output Widget
widget = ipywidgets.VBox(children = [heading, view])
```

Below are my versions of packages.

> itkwidgets 1.0a16 pypi_0 pypi
> ipywidgets 7.6.5 pyhd3eb1b0_1 defaults

When running with version 0.X of `itkwidgets` this error does not occur

> itkwidgets 0.32.0 py38h32f6830_0 conda-forge

**Comment by thewtex** [comment]
@kevinsweeney84 thanks for the report.

In version 1.X, we also work outside Jupyter.

We are adding detection of the environment in #558 

For Jupyter environments, we will:

- Create a new ViewerWidget class that inherits from `ipywidgets.Widget` -- return this from `view`.
- Hook up callback from the viewer events to update associated traitlets
- Have traitlets call `get_` methods to provide their value

**Comment by kevinsweeney84** [comment]
@thewtex I see that you have closed #558 , congrats.
Do you have an approximate timeline as to when this will be available through pip?

**Comment by thewtex** [comment]
Yes, making progress towards this. The other blocker is #568 , which is a bit tricky and hard to estimate a timeline.

**Comment by kevinsweeney84** [comment]
@thewtex I installed your commit from #558 using pip
- `pip install git+https://github.com/InsightSoftwareConsortium/itkwidgets.git@302126a3e17d3ef8b848136a0c973e0bb142ceee#egg=itkwidgets`
- Installed package information
  - `itkwidgets                1.0a18.dev12+g302126a          pypi_0    pypi`

but I am still seeing the same error:
- `TraitError: The 'children' trait of a VBox instance contains an Instance of a TypedTuple which expected a Widget, not the Viewer at '0x2aab2b843430'.`

Should this be fixed in this instance?

**Comment by thewtex** [comment]
@kevinsweeney84 not, yet but we are working on it.

**Comment by hjmjohnson** [comment]
@thewtex   This bug is affecting itk notebooks tutorials as well.
